benchmarks/graphics_pipeline/GraphicsBenchmarkApp.h

```diff
@@ -414,6 +414,20 @@ class GraphicsBenchmarkApp
     std::array<grfx::ShaderModulePtr, kFullscreenQuadsTypes.size()>      mQuadsPs;
 
     BlitContext mBlit;
+    // Metrics Data
+    struct MetricsData
+    {
+        enum MetricsTypes : size_t
```

Comment:
nit: I think it should be MetricType since each value represents a single type, but feel free to ignore me here if you think otherwise

Reply:
fixed